shardingsphere-infra/shardingsphere-infra-binder/src/main/java/org/apache/shardingsphere/infra/binder/segment/select/projection/impl/AggregationProjection.java

```diff
@@ -45,7 +43,8 @@ public class AggregationProjection implements Projection {
     
     private final String innerExpression;
     
-    private final String alias;
```

Comment:
Why remove final modifier?

Reply:
In general, "alias" was  initialized by constructor method. But when AggregationProjection belong to ExpressionProjection, I want to reset AggregationProjection's alias.